src/main/java/Parser.java

```diff
@@ -0,0 +1,34 @@
+public class Parser {
+    public static Command parse(String input) throws InvalidArgumentException, InvalidCommandException {
+            String[] commands = input.split(" ", 2);
+            Commands current;
+            try {
+                current = Commands.valueOf(commands[0].toUpperCase());
+                switch(current) {
```

Comment:
Correct indentation for switch method, this is a common mistake. Good job 👍  !

Reply:
thanks :)